styles/page_interfaces.less

```diff
@@ -163,6 +186,22 @@
         padding: 0.5em;
         .solid-border;
         .page-interface .text-pane;
+        .proofingfont {
+            input {
+                background-color: white;
+                color: black;
+                padding: 3px;
+                .x-thin-border(@border-color-base-lowc);
+            }
+        }
+    }
+    #wc_header {
+        input, select {
+            background-color: @page-background;
+            color: @page-text;
+            .solid-border;
+            .button;
```

Comment:
I don't think we want the button style on the drop-down -- just on the button. I also think we want to tweak the `.button` class so it looks more like our other buttons (now that we set styles for buttons). Consider:
```diff
diff --git a/styles/global.less b/styles/global.less
index 1fd54d2b1..58f8d253d 100644
--- a/styles/global.less
+++ b/styles/global.less
@@ -116,20 +116,6 @@
     visibility: hidden;
 }

-.button {
-    /*
-     * We can't use appearance: button here because Chrome has deprecated it:
-     * https://developers.google.com/web/updates/2019/12/chrome-80-deps-rems
-     * so we make a "button like object" that won't really resemble the other
-     * buttons rendered, but still looks like a button.
-     */
-    border: solid 1px lightgray;
-    border-radius: 5px;
-    font-size: 0.7rem;
-    padding: 2px 6px;
-    cursor: default;
-}
-
 .nodisp {
     display: none;
 }
diff --git a/styles/layout.less b/styles/layout.less
index d8db95956..0b9c1647e 100644
--- a/styles/layout.less
+++ b/styles/layout.less
@@ -98,7 +98,21 @@
 }

 .button {
+    /*
+     * We can't use appearance: button here because Chrome has deprecated it:
+     * https://developers.google.com/web/updates/2019/12/chrome-80-deps-rems
+     * so we make a "button like object" that won't really resemble the other
+     * buttons rendered, but still looks like a button.
+     */
+    border-radius: 3px;
+    cursor: default;
     .sans-serif;
+    .solid-border;
+}
+
+span.button {
+    font-size: 0.8em;
+    padding: 1px 4px;
 }

 /* ------------------------------------------------------------------------ */
diff --git a/styles/page_interfaces.less b/styles/page_interfaces.less
index aaf2e5a97..6d88d7252 100644
--- a/styles/page_interfaces.less
+++ b/styles/page_interfaces.less
@@ -200,6 +200,8 @@
             background-color: @page-background;
             color: @page-text;
             .solid-border;
+        }
+        button, input[type=submit], input[type=button] {
             .button;
         }
     }
```

Reply:
Thanks, done.

Button styling moved from `global.less` to `layout.less`, and styling tweaked.

Fair point about not styling the dropdown as a button. That was sloppy of me. Turns out that FF, Chrome and Safari all have different default button looks, and I was trying to match what I was seeing in FF (both the drop-down and the button have a border radius). The dropdown no longer uses the button styling.